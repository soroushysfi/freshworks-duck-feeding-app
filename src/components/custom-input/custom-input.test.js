import { shallow } from 'enzyme'
import CustomInputComponent from './custom-input.component'

describe("CustomInput Component tests", () => {
it('expect to match CustomInputComponent snapshot', () => {
    expect(shallow(<CustomInputComponent />)).toMatchSnapshot()

})
})