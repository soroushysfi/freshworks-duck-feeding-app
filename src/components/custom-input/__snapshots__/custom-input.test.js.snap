// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomInput Component tests expect to match CustomInputComponent snapshot 1`] = `
<div
  className="form-input-material"
>
  <label>
    : 
  </label>
  <input
    className="form-control-material"
  />
</div>
`;
